blobit-core/src/main/java/org/blobit/core/cluster/BookKeeperBlobManager.java

```diff
@@ -471,8 +473,11 @@ void scanAndDeleteLedgersForBuckets(List<BucketMetadata> buckets) throws ObjectM
     }
 
     boolean dropLedger(long idledger) throws ObjectManagerException {
+        LOG.log(Level.INFO, "Active writers in dropledger = {0}", activeWriters);
```

Comment:
please remove this debug, it will be very annoying in production

Reply:
Yes I will remove it, they were debugging to understand why the tests failed